Gemfile

```diff
@@ -128,7 +128,7 @@ end
 
   gem 'useragent'  # detect browser types
 
-  gem 'react-rails', '~> 2.3.1' # React 15.6.2
+  gem 'react-rails', '~> 2.6.1' # React 16.9.0
```

Comment:
What is the gem used for now? I was thinking that react would be bundled with the portal-pages library.

Reply:
Doug and I talked about this directly, and the plan is to leave the dependency in the portal until at least after the portal-pages code has been brought back into the portal codebase.